website/content/en/docs/deployment/cognito/manifest/guide.md

```diff
@@ -125,45 +162,60 @@ make deploy-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=cognito
 {{< /tab >}}
 {{< /tabpane >}}
 
-## 5.0 Updating the domain with ALB address
+## 5.0 Update the domain with the ALB address
+
+1. Check if the ALB is provisioned. This may take a few minutes.
+
+    ```bash
+    kubectl get ingress -n istio-system
+    Warning: extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
+    NAME            CLASS    HOSTS   ADDRESS                                                              PORTS   AGE
+    istio-ingress   <none>   *       k8s-istiosys-istioing-xxxxxx-110050202.us-west-2.elb.amazonaws.com   80      15d
+    ```
+    If `ADDRESS` is empty after a few minutes, see [ALB fails to provision]({{< ref "/docs/troubleshooting-aws.md#alb-fails-to-provision" >}}) in the troubleshooting guide.
+
+1. When the ALB is ready, copy the DNS name of the load balancer and create a CNAME entry for it in Route53 under the subdomain (`platform.example.com`) for `*.platform.example.com`
+
+    ![subdomain-*.platform-and-*.default-records](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/subdomain-*.platform-and-*.default-records.png)
+
+1. Update the type `A` record created for `platform.example.com` using ALB DNS name. Change from `127.0.0.1` to the ALB DNS name. You have to use the alias form under `Alias to application and classical load balancer` then select a region and your ALB address.
+
+    ![subdomain-A-record-updated](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/subdomain-A-record-updated.png)
 
-1. Check if ALB is provisioned. This may take a few minutes.
-    1. ```bash
-        kubectl get ingress -n istio-system
-        Warning: extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
-        NAME            CLASS    HOSTS   ADDRESS                                                              PORTS   AGE
-        istio-ingress   <none>   *       k8s-istiosys-istioing-xxxxxx-110050202.us-west-2.elb.amazonaws.com   80      15d
+1. Here is a screenshot of all the record sets in the hosted zone for reference.
+
+    ![subdomain-records-summary](https://raw.githubusercontent.com/awslabs/kubeflow-manifests/main/website/content/en/docs/images/cognito/subdomain-records-summary.png)
+
+## 6.0 Connect to the central dashboard
+
+1. The central dashboard should now be available at `https://kubeflow.platform.example.com`. 
+Before connecting to the dashboard:
+
+    * Go to the Cognito console and create some users in `Users and groups`. These are the users who will log in to the central dashboard.
```

Comment:
curious: any reason to change many of the numbering to bullet points? Arent these still sub steps of this parent step?

Reply:
Because it was becoming more noisy than helpful. 
Especially when you have a numbered list which itself has a numbered list etc... I often find that less is more. Our users are smart and will get the point.
Personal opinion. 

I am happy to discuss or change back to the original version.